src/mapml/handlers/ContextMenu.js

```diff
@@ -723,13 +775,135 @@ export var ContextMenu = L.Handler.extend({
         L.DomEvent.stop(e);
         this._focusOnLayerControl();
       }
+    } else if (key === 38) { //up arrow
+      if (!this._coordMenu.hasAttribute('hidden') && 
+        (document.activeElement.shadowRoot == null || //null happens when the focus is on submenu and when mouse hovers on main menu, submenu disappears
```

Comment:
You should change all the "==" operators to "===", they are quite different! 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality
vs
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality

Reply:
I fixed it, because the test with the Escape key wasn't working due to the == operator